arelle/plugin/validate/NL/rules/fr_kvk.py

```diff
@@ -120,6 +120,36 @@ def rule_fr_kvk_2_03(
                     )
 
 
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[
+        DISCLOSURE_SYSTEM_NT16,
+        DISCLOSURE_SYSTEM_NT17,
+    ],
+)
+def rule_fr_kvk_2_02(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation] | None:
+    """
+    FR-KVK-2.02: The attribute 'xml:lang' MUST contain the same value within an XBRL instance document
+    """
+    modelXbrl = val.modelXbrl
+    doc_xml_lang = []
+    for fact in modelXbrl.facts:
+        if fact.xmlLang and fact.xmlLang not in doc_xml_lang:
+            doc_xml_lang.append(fact.xmlLang)
```

Comment:
This should presumably also check footnotes and the lang attribute of the containing xbrl element. Also, Arelle is written with camel case variables, not the typical Python snake case.